### DISCUSSION TITLE
你也喜歡熬夜嗎?

### DISCUSSION THREAD

**Discussion by Hsucuchia** [1101-GS4719-C/Python-Community-GS4719-C#1301] [General]
每天都有做不完的事情，一不小心又熬夜了
聽說熬夜會變笨? 睡眠越少處理事情時越容易出錯，疲勞也會讓自己專注力降低
熬夜還會變胖? 熬夜好累一點都不想動，沒有動就沒有熱量消耗，自然就會變胖
總是忍不住熬夜?雖然熬夜缺點很多，但為什麼我們還是要熬夜?
1. 很多代辦事項還沒完成: 每天回到宿舍已經很晚，但還是好多事還沒完成，只能犧牲睡覺時間
2. 想要掌握自己的時間: 白天總感覺沒有屬於自己的休息時間，到了晚上，就會想把握難得的自由時間
3. 手機太好玩: 想要放鬆一下，滑手機、看看劇、打遊戲、看小說，結果不知不覺就天亮了

**Comment by brian431** [comment]
其實我熬夜大部分時候不是因為有事情要做或是想完，而是下午或晚上的時候不小心小睡個一小時之類的。導致真正該睡覺的時候根本睡不著，然後一直循環養成這種作息。一時之間要改掉並不簡單，但其實主要是因為也沒太大動力去養成正常點的作息。

**Comment by CCello** [comment]
熬夜的原因可能有很多，例如工作壓力、學業壓力、娛樂、社交等等。但無論是什麼原因，都不建議經常熬夜。如果你經常需要熬夜，可以試著找到解決辦法，例如改變工作或學習方式、安排更好的時間管理、找到其他的娛樂或社交方式等等。

**Comment by tower20021202** [comment]
雖然我有時在考前熬夜是為了抱佛腳，但大部分的時候，會熬夜只不過是想要再多滑一下手機，多看一部影集又或是在打一場遊戲，往往１２點前就能入睡，拿起手機不知不覺就２點了。因此我現在每次在快１２點時就會設定一個鬧鐘，提醒自己該睡叫了，並且把手幾拿到我看不到的地方趕快睡覺，這樣做我個人實驗下來認為效果還不錯

**Comment by Jack-1D** [comment]
之前高三升學時每天都讀書讀到晚上只睡三個半到四小時。不只上課變得很想睡覺，記憶力也變得很差，連思考都會變得很遲鈍。所以升上大學後，我很珍惜睡眠時間，如果沒有重要的考試，都會睡到七小時。睡前頂多休息看手機一小時，就會很克制的放下手機。

**Comment by lyien1021** [comment]
我的話理智上都會告誡自己不能熬夜，但我是一個有蠻嚴重的拖延症、而事情沒做完又無法安心睡著的人，導致我常常都無法準時睡覺😢，我覺得遇到這種狀況我會逼自己去圖書館或k書中心，感受那邊的氛圍做事效率比較高。

**Comment by YoYooHuang** [comment]
其實我覺得熬夜都是沒必要的。如果仔細思考自己的時間分配，會發現其實時間都是夠的。但都會因為腦中想著：也才幾分鐘，也做不了什麼事情，來滑一下手機好了。之類的想法，導致一些瑣碎的時間浪費掉，但一整天下來可能可以蒐集到一個小時甚至更多，如果我們好好運用這些零碎的時間，就能加強自己的生產力！而且熬夜這件事情也是在消耗自己的本錢，只會讓身體越來越差，進入無限的惡性循環。

**Comment by orianna901001** [comment]
我常常覺得自己好像在熬夜，固定太晚睡似乎也是也是一種熬夜，但就是心裡覺得不捨得睡，雖然一整天時間24小時一定是夠的，但就是想"把握這個當下" 。

**Comment by ziihui** [comment]
戒不掉熬夜><在深夜靜謐的夜晚，大腦通常會處於一種比較放鬆、沒有干擾的狀態，有時候會有更多的靈感和創意湧現出來，所以我很喜歡在晚上動腦(? ，但缺點是隔天起床真的好累><

**Comment by sjh501** [comment]
在大學階段要避免熬夜頗難，我個人上個學期段考周幾乎是顛倒著過，結果就是考試成績全部炸裂。後來只要熬夜看到時間都會有種罪惡感。這學期修更多學分，依舊經常熬夜，但在比較不忙碌的幾天試著多運動流汗早點上床，隔天也能早點離開床舖做事其實感覺也是不錯。一早走出房間看到外面的陽光灑進來，那種心情愉悅讓我一早做事的效率都好很多，也發現有些事如果早些睡隔天一早再處理是不錯的選擇。

**Comment by Jamesmoneya** [comment]
熬夜是天性使然，因為大部分的人都有拖延症，不到最後關頭是不會善罷甘休的，為了明天的課業可以順利完成，不是選擇提早完成，而是選擇在結束的前一兩個小時，用你此生最有效率的放法完成他，其實會熬夜的人效率才是最高的xd

**Comment by llh26** [comment]
其實我自己熬夜的最大主因是「愛玩」，我平時真的很常因為滑手機而晚睡，時間咻咻咻地就過了一大半，其實原因跟文章中的第二點一樣，難得一整天下來終於有時間可以做自己想做的事情，我就想要做到我累了再睡，有種太早睡太可惜了的感覺，真的希望自己可以改掉這個壞習慣⋯。

**Comment by Evening218** [comment]
現代人幾乎都有明明已經該休息了但還是繼續熬夜的習慣,我們一般將這個狀況稱為睡眠拖延症,也就是向上面的留言所說:終於有時間做自己想做的事。我自己改善睡眠拖延症的方法是安排好工作和娛樂的時間,並且每天都給明天安排至少一件值得期待的事情,不要讓自己覺得捨不得結束今天

**Comment by RogelioKG** [comment]
有時候我是不知不覺的熬夜，
我會想說：再做一點，再做一點，再做一點......
結果一點一點，一小時就這樣過去了。
這確實是時間掌控的問題，
事情有多進展一點沒錯，
但這只不過是用白天的睡眠時間換來的。
最近都有在反省這件事情，
認為自己不如早睡早起做事還比較好。
因為早起的習慣可以迎合早八的課，這是有優點的。

**Comment by poyen0806** [comment]
老實說，我熬夜是因為在半夜打code更能集中注意力。
這是有說法可以證明的。

1. 因為半夜大家都睡了，所以比較安靜，不會有人打擾
2. 因為已經半夜了，沒有多餘的精神可以去分心

例如說如果要我做一個集中注意力的計時器程式
我一定要在半夜的時候才會知道要用tkinter來做GUI
或是用pygame讓介面更鮮明

**Comment by lichilun** [comment]
我懂那種感覺，明知道熬夜不好還是熬夜.....
對我而言我熬夜的可能性如下：
1.每次到期中、末當壓線戰士，考前暴讀一波，直接拼到通霄用2天讀完，半學期得課程。
2.手機誘惑太大了（這我），本來只是想放鬆一下，結果一滑就停不下來。尤其那種（再看一集就睡）的心態，最後永遠都是（再來最後一集）。

**Comment by yurukung** [comment]
喜灣

**Comment by SendaiHatsuki** [comment]
我覺得現代人熬夜應該是環境使然，平日要做的事情真的太多了，像是讀書、報告、工作等等，而晚上等大部份事情都處理完才有自己的時間做自己喜歡的事情，所以才會熬夜。

**Comment by a414414** [comment]
喜歡兒

**Comment by Kaz9487** [comment]
不喜歡吧。我每天都會想我是不是不該熬夜，都想著要1:30左右就入眠。
但總是東西做一做，刷個牙，在床上滑一下手機就要3:00了
不想熬夜但就一直，可能我需要提升一下自我控制能力也說不定
